libraries/Microsoft.Bot.Connector/Authentication/BotFrameworkAuthentication.cs

```diff
@@ -59,6 +59,26 @@ public virtual BotFrameworkClient CreateBotFrameworkClient()
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Gets the originating audience from Bot OAuth scope.
+        /// </summary>
+        /// <returns>The originating audience.</returns>
+        public virtual string GetOriginatingAudience()
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Validate Bot Framework Protocol requests.
```

Comment:
Update the documentation to be more descriptive (this method is to be used for authenticating Skill requests)